spec/rspec/matchers/built_in/respond_to_spec.rb

```diff
@@ -33,6 +33,12 @@ def obj.foo(a, *args); end
     expect(obj).to respond_to(:foo).with(1).argument
   end
 
+  it "passes if target is new" do
```

Comment:
This doc string is overly general.  

> expect(...).to respond_to(:sym).with(1).argument passes if target is new

That reads like the expectation will always pass if the target is `new`, regardless of how `initialize` is defined.

Reply:
I agree, reworded